### PR TITLE
Added lecturer comparison view.

### PR DESCRIPTION
**Pull request by povilassta**

### CODE REVIEW

File: Controller/Lecturer/ILecturerController.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿using Advisor.View;
```
Comment (Gintasp) [inline Controller/Lecturer/ILecturerController.cs]: Nereikalingas using

File: Controller/Lecturer/ILecturerController.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+namespace Advisor.Controller
+{
+    public interface ILecturerController
```
Comment (Gintasp) [inline Controller/Lecturer/ILecturerController.cs]: `ILecturerComparisonController`, lecturer controller skirtas lecturer formai rodyt yra/bus

File: Controller/Lecturer/LecturerController.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿using Advisor.View;
+using Advisor.Model;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace Advisor.Controller
+{
+    public class LecturerController : ILecturerController
```
Comment (Gintasp) [inline Controller/Lecturer/LecturerController.cs]: `LecturerComparisonController`

File: Controller/Lecturer/LecturerController.cs [hunk 4]
```diff
@@ -0,0 +1,17 @@
+﻿using Advisor.View;
```
Comment (Gintasp) [inline Controller/Lecturer/LecturerController.cs]: Nereikalingus using metam lauk.

File: View/Comparison/LecturerComparisonView.cs [hunk 5]
```diff
@@ -0,0 +1,34 @@
+﻿using System;
```
Comment (Gintasp) [inline View/Comparison/LecturerComparisonView.cs]: Nereikalingu using kruva

File: View/Comparison/LecturerComparisonView.cs [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    {
+        public ILecturerController LecturerController { get; set; }
+        public LecturerComparisonView(LecturerController lecturerController)
+        {
+            LecturerController = lecturerController;
+            InitializeComponent();
+
+        }
+
+        private void Lecturer1_Click(object sender, EventArgs e)
```
Comment (Gintasp) [inline View/Comparison/LecturerComparisonView.cs]: `OnLecturerOneSelectClick`

File: View/Comparison/LecturerComparisonView.cs [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+        {
+            LecturerController.DisplayPopup();
+        }
+
+        private void Lecturer2_Click(object sender, EventArgs e)
```
Comment (Gintasp) [inline View/Comparison/LecturerComparisonView.cs]: `OnLecturerTwoSelectClick`

File: View/PopupLecturerComparisonView.cs [hunk 8]
```diff
@@ -0,0 +1,20 @@
+﻿using System;
```
Comment (Gintasp) [inline View/PopupLecturerComparisonView.cs]: Kruva nereikalingu using

**Comment by povilassta** [comment]
Nu ryt sutvarkysiu nx

**Pull request merged**